lib/ClusteredMapView.js

```diff
@@ -168,6 +169,7 @@ const ClusteredMapView = ({
               onPress={_onClusterPress(marker)}
               clusterColor={clusterColor}
               clusterTextColor={clusterTextColor}
+              tracksClusterViewChandges={tracksClusterViewChandges}
```

Comment:
Wrong naming, it should be: tracksClusterViewChanges.

Reply:
Why is `tracksClusterViewChanges` assigned to 'react-native-maps' Marker? Should it not be `tracksViewChanges={tracksClusterViewChandges}`